pkg/kube/nsmounter

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+PROC_DIR="/proc"
+
+os_distro_talos="talos"
```

Comment:
Is this from an existing file with talks support and you're adding eve support to it? If so can we upstream the eve support?

Reply:
yes the upstream work is going to be submitted soon